install/kubernetes/templates/service.yaml

```
@@ -13,7 +14,7 @@ metadata:
 spec:
   ports:
     - name: metrics
-      port: 2112
+      port: {{ .Values.tetragon.prometheus.port }}
       protocol: TCP
       targetPort: 2112
```

From https://github.com/cilium/tetragon/blob/5c3fd60f10c72d7854a287f877985c49eb6f737c/install/kubernetes/templates/tetragon_configmap.yaml#L34-L37 I gather that `.Values.tetragon.prometheus.port` should be used as the `targetPort` rather than the svc port.

```suggestion
      port: 2112
      protocol: TCP
      targetPort: {{ .Values.tetragon.prometheus.port }}
```

---

You're totally right. Already fixed!